src/callbacks.cpp

```diff
@@ -569,32 +690,6 @@ void MissionControl::health_callback(const interfaces::msg::UAVHealth &msg) {
                       msg.is_magnetometer_calibration_ok &&
                       msg.is_local_position_ok && msg.is_global_position_ok &&
                       msg.is_home_position_ok && msg.is_armable;
-
-    // Check that drone health is still good
```

Comment:
I removed this section because the fcc bridge takes care of the drone health and I only use this check before I arm.

@J0las and I decided to split the responsibilities: @J0las cares about the drone and its health, while I care about the mission and the other nodes.
I thought that duplicating the effort just leads to problems.